### ISSUE TITLE
SPF causes email delivery problems

### ISSUE THREAD

**Issue by Kennric** [osuosl/formsender#80]
If a form submitter's email address is in a domain with SPF, emails 'FROM' that address but sent by formsender are rejected.

**Comment by ramereth** [comment]
Looking at [email headers](https://support.osuosl.org/Ticket/Attachment/WithHeaders/2138188) from when we had this in a Drupal webform, it had this in there:

```
Sender: info@osuosl.org
```

According to [RFC 4021](https://tools.ietf.org/html/rfc4021#section-2.1.2), the header is defined as:

> Specifies the mailbox of the agent responsible for the actual transmission of the message.

That should tell Gmail and others to check our SPF settings instead of email host we have set in the ``From:`` header.

I imagine if we add this to the field with a configurable email address, this should resolve the issue.

**Comment by Kennric** [comment]
This should be a simple fix, add an `email_sender` setting to the config file, and add

```
msg_send['Sender'] = conf.EMAIL[email_sender]
```

To request_handler.py in the send_email method.

**Comment by Kennric** [comment]
resolved by pr #81

**Comment by knightsamar** [comment]
Is this ready to deploy on production ?
